CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ Ansible Changes By Release
 ### Major Changes
 
 ### Deprecations (to be removed in 2.10)
+* `oif_prefix` and `oif_source` properties are deprecated in the nxos_igmp_interface
```

Comment:
Minor suggestion, start with the module name, will make it easier to read:
`* nxos_igmp_interface`